index.js

```diff
@@ -112,6 +122,7 @@ async function run(context, plugins) {
       if (options.dryRun) {
         logger.warn(`Skip ${nextRelease.gitTag} tag creation in dry-run mode`);
       } else {
+        await fetchNotes(options.repositoryUrl, {cwd, env});
```

Comment:
Could you add a comment on why we fetch the notes here, with a reference to the pull request?